app/controllers/expenses_additional_sources_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+class ExpensesAdditionalSourcesController < StandardStepsController
+  private
+
```

Comment:
we've been defining the `existing_attributes` method to ensure that attrs are filled in on the page when someone returns. Is that functionality working here? Perhaps a controller spec would be a good way to confirm